### PR TITLE
chore: bump kubo and pubsub to pull in validator updates (#23

### PR DESCRIPTION
**Pull request by smrz2001**

### CODE REVIEW

**Comment by stbrody** [comment]
is there any way we can test/verify that this build actually has the validator running properly?

**Comment by stbrody** [comment]
maybe attach a debugger and put a breakpoint in the validator code somewhere?

**Comment by smrz2001** [comment]
> is there any way we can test/verify that this build actually has the validator running properly?

> maybe attach a debugger and put a breakpoint in the validator code somewhere?

Can't do this with the Docker image - I'll do it with Kubo directly with a debugger attached, @stbrody.

File: Dockerfile [hunk 1]
```diff
@@ -3,7 +3,7 @@ FROM golang:1.19.1-buster as clone
 
 WORKDIR /clone
 
-RUN git clone --depth 1 --branch v0.18.1 https://github.com/ipfs/kubo kubo
+RUN git clone --depth 1 --branch v0.19.1 https://github.com/ipfs/kubo kubo
```
[the remark this answers is not in the document]
Comment (smrz2001) [reply Dockerfile:6]: It should be compatible. The dependency conflicts were if we stayed on v0.18.1 that's why I updated to v0.19.1.

I still need to pull in vyzo's change to kubo so this PR isn't complete ☹️

Might just make changes to a Kubo fork then pull that commit in directly.

**Pull request merged**